cmd/storageserv/config/config.go

```diff
@@ -163,7 +165,7 @@ func LoadConfig(ssConfigFile string) (err error) {
 	if !serverConfig.MicroShardsEnabled && len(serverConfig.DB.DbPaths) >= 1 {
 		tagFile := fmt.Sprintf("%s/microshard_enabled.txt", serverConfig.DB.DbPaths[0].Path)
 		_, err := os.Stat(tagFile)
-		if !os.IsNotExist(err) {
+		if errors.Is(err, fs.ErrNotExist) {
```

Comment:
Missing ! operator in if stmt.

Reply:
Add ! operator in if statement in latest push , rebased